docs/documentation/development/bindings/thing-handler.md

```diff
@@ -180,6 +180,27 @@ Furthermore bindings can specify a localized description of the thing status by
 rate_limit=Device is blocked by remote service for {0} minutes. Maximum limit of {1} configuration changes per {2} has been exceeded. For further info please refer to device vendor.
 ```
 
+## Offering a callback URL
+
+Some things might require a `callback` URL which should be bound to a certain network interface. A user can configure his default network address via Paper UI under `Configuration -> System -> Network Settings`. To obtain this configured address the `ThingHandlerFactory` needs a `service reference` to the `NetworkInterfaceService` in its `OSGI-INF/MyHandlerFactory.xml`:
```

Comment:
Please add a new line for every sentence - this makes diffing and reviews easier.

The correct name seems to be `NetworkAddressProvider`.
The example is not good, because we by now ask people to use DS annotations, not XML files.